tests/cli.test.ts

```diff
@@ -133,83 +133,65 @@ describe('ember addon <the addon> -b <this blueprint>', () => {
         expect(exitCode).toEqual(0);
       });
     });
-  });
 
-  describe('--typescript', () => {
-    let cwd = '';
-    let tmpDir = '';
-    let distDir = '';
+    describe(`--typescript with ${packageManager}`, () => {
```

Comment:
Basically just white-space changes here due to putting this onto previous `describe()` that runs for every package manager. Best reviewed with `Hide whitespace` enabled!